colossalai/zero/init_ctx/init_context.py

```diff
@@ -88,6 +88,7 @@ class ZeroContextConfig(object):
     """The configuration used to control zero context initialization.
 
     Args:
+        target_device (torch.device): The device where param data after exiting the context.
```

Comment:
The device where param data are after exiting the context.

Reply:
done